### ISSUE TITLE
XML type error while renewing

### ISSUE THREAD

**Issue by Janoz-NL** [Grexxboxx/docker-certbot-transip#1]
After running fine for a while I noticed my certificates weren't renewed. Got the following error. THis might be the same issue as https://github.com/hsmade/certbot-dns-transip/issues/19 

```
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Processing /etc/letsencrypt/renewal/domain.tld.conf
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Cert is due for renewal, auto-renewing...
Plugins selected: Authenticator certbot-dns-transip:dns-transip, Installer None
Renewing an existing certificate
Performing the following challenges:
dns-01 challenge for domain.tld
dns-01 challenge for domain.tld
Cleaning up challenges
Encountered exception during recovery:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/error_handler.py", line 124, in _call_registered
    self.funcs[-1]()
  File "/opt/certbot/src/certbot/auth_handler.py", line 220, in _cleanup_challenges
    self.auth.cleanup(achalls)
  File "/opt/certbot/src/certbot/plugins/dns_common.py", line 77, in cleanup
    self._cleanup(domain, validation_domain_name, validation)
  File "/usr/local/lib/python2.7/site-packages/certbot_dns_transip/dns_transip.py", line 70, in _cleanup
    self._get_transip_client().del_txt_record(domain, validation_name, validation)
  File "/usr/local/lib/python2.7/site-packages/certbot_dns_transip/dns_transip.py", line 89, in _get_transip_client
    return _TransipClient(username=username, key_file=key_file)
  File "/usr/local/lib/python2.7/site-packages/certbot_dns_transip/dns_transip.py", line 97, in __init__
    self.domain_service = DomainService(login=username, private_key_file=key_file)
  File "/usr/local/lib/python2.7/site-packages/transip/service/domain.py", line 12, in __init__
    super(DomainService, self).__init__('DomainService', *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/transip/client.py", line 73, in __init__
    self.soap_client = SudsClient(self.url, doctor=doc, **suds_kwargs)
  File "/usr/local/lib/python2.7/site-packages/suds/client.py", line 122, in __init__
    sd = ServiceDefinition(self.wsdl, s)
  File "/usr/local/lib/python2.7/site-packages/suds/servicedefinition.py", line 58, in __init__
    self.addports()
  File "/usr/local/lib/python2.7/site-packages/suds/servicedefinition.py", line 86, in addports
    method = (m.name, binding.param_defs(m))
  File "/usr/local/lib/python2.7/site-packages/suds/bindings/rpc.py", line 35, in param_defs
    return self.bodypart_types(method)
  File "/usr/local/lib/python2.7/site-packages/suds/bindings/binding.py", line 379, in bodypart_types
    raise TypeNotFound(query.ref)
TypeNotFound: Type not found: '(array, http://www.w3.org/2001/XMLSchema, )'
Attempting to renew cert (domain.tld) from /etc/letsencrypt/renewal/domain.tld.conf produced an unexpected error: Type not found: '(array, http://www.w3.org/2001/XMLSchema, )'. Skipping.
```

**Comment by bartlaarhoven** [comment]
Yes, this seems like the same issue.

**Comment by Janoz-NL** [comment]
Made a pull request that fixes this issue #2

**Comment by bartlaarhoven** [comment]
Thank you. Merged the PR.
